torchrl/_utils.py

```diff
@@ -529,3 +531,39 @@ def clone(self):
 def get_trace():
     """A simple debugging util to spot where a function is being called."""
     traceback.print_stack()
+
+
+def unravel_key_list(key_list):
+    """Temporary fix for change in behaviour in unravel_key_list."""
+    if isinstance(key_list, str):
+        raise TypeError("incompatible function arguments")
+    key_list_out = []
+    for key in key_list:
+        key = unravel_keys(key)
+        if isinstance(key, tuple) and len(key) == 1:
+            key_list_out.append(key[0])
+        else:
+            key_list_out.append(key)
+    return key_list_out
+
+
+def unravel_keys(key):
```

Comment:
Why are we still keeping the name unravel_keys? Cant we do all this and also stop using that plural in all the 2 projects?
I thought after this whole thing we were going to have:

- unravel_key
- unravel_key_list
- _unravel_tuple_key

with unravel_keys dead

Reply:
this is just for this PR, it'll be used for 1 hour max
I just don't want the tests to break at any time bc that will be a pain to handle internally on fbcode
I can rename it unravel_key but that's beyond the point IMO
We could name it `_tmp_fun_that_we_will_remove_` fwiw

Reply:
ok got it go for it

Reply:
btw we won't kill anything for bc reasons, we'll just keep both names (perhaps wrapping the plural in a deprecation decorator)